xbmc/GUIInfoManager.cpp

```diff
@@ -5530,6 +5532,18 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
       return letter;
     }
     break;
+  case LISTITEM_TAG:
+    if (item->HasVideoInfoTag())
+    {
+      return StringUtils::Join(item->GetVideoInfoTag()->m_tags, g_advancedSettings.m_videoItemSeparator);
+    }
```

Comment:
you couls leave it as it is, because we have no strict coding  guideline for it. someone prefer this someone the other way :)